src/components/RadioButton.js

```diff
@@ -1,38 +1,88 @@
+// ***NEED TO FIGURE OUT HOW TO SWITCH ON AND OFF AFTER SELECTION***
```

Comment:
Note:  Since you already merged my PR, you will have to pull from the remote again.  There might be more merge conflicts.  Also, make sure to npm install to grab some other packages.